lib/snip.c

```diff
@@ -436,6 +436,20 @@ grn_snip_get_delimiter_regexp(grn_ctx *ctx, grn_obj *snip, size_t *length)
   return snip_->delimiter_pattern;
 }
 
+static grn_rc
+grn_snip_expand_cond(grn_ctx *ctx, grn_snip *snip)
+{
+  size_t new_cond_capacity = snip->cond_capacity * 2;
+  snip_cond *new_cond = (snip_cond *)GRN_REALLOC(snip->cond,
+                                          sizeof(snip_cond) * new_cond_capacity);
```

Comment:
Indentation is broken.

```suggestion
  size_t new_cond_size = sizeof(snip_cond) * new_cond_capacity;
  snip_cond *new_cond = (snip_cond *)GRN_REALLOC(snip->cond, new_cond_size);
```

Reply:
Right, I have fixed it.
(Sorry for my carelessness)